packages/babel-plugin-relay/createModernNode.js

```diff
@@ -125,12 +125,12 @@ function getRelativeImportPath(
   const relative = path.relative(
     path.dirname(filename),
     path.resolve(artifactDirectory),
-  );
+  ).replace(/\\/g, '/');
```

Comment:
@martinbooth  In windows, it gives two backslashes so I think  the regex will be
`replace(/\\\\/g, '/')`

Reply:
Maybe you're looking at the output of evaluating that expression in node? That escapes the output: i.e.
```
> path.relative(path.dirname("C:\\dir1\\dir2\\dir3"), path.resolve("C:\\"))
'..\\..'
```
If you print it to the console unescaped you'll see that there is just one backslash:
```
> console.log(path.relative(path.dirname("C:\\dir1\\dir2\\dir3"), path.resolve("C:\\")))
..\..
```